### ISSUE TITLE
Have you run your code?

### ISSUE THREAD

**Issue by le270** [niecongchong/DANet-keras#1]
Thanks for your contribution! And I hope that you could provide the result of your work.

**Comment by niecongchong** [comment]
I'm sorry, I'm doing aerial image segmentation, so I don't have an experiment to replicate the danet paper. You can try it yourself.
I very much hope to be able to maintain communication.

**Comment by le270** [comment]
So how did you get the training process?(I mean the epoch loss and the picture)

**Comment by niecongchong** [comment]
On the one hand, this is related to the aerial image data set I have been using, on the other hand, it is also more important, more attempts on the tuning technique

**Comment by le270** [comment]
I haven't see the code that "from keras.layers.attention import PAM, CAM".And I can't search any information about it. Could you tell me why?Would you write the model code for fun,and you didn't test  the model code you write?

**Comment by niecongchong** [comment]
First of all, it's ‘from layers.attention import PAM, CAM’，not what you said "from keras.layers.attention import PAM, CAM".
Secondly, the definition of PAM and CAM is in /DANet-keras/layers/attention.py.
Third, be polite !!!

**Comment by niecongchong** [comment]
@le270 
First of all, it's ‘from layers.attention import PAM, CAM’，not what you said "from keras.layers.attention import PAM, CAM".
Secondly, the definition of PAM and CAM is in /DANet-keras/layers/attention.py.
Third, be polite !!!

**Comment by le270** [comment]
I am very sorry. I turns out that is my fault. I don't have  see  the code clearly. Thanks  for your contribution.
